exercises/practice/nth-prime/.meta/config.json

```diff
@@ -15,5 +15,5 @@
   },
   "blurb": "Given a number n, determine what the nth prime is.",
   "source": "A variation on Problem 7 at Project Euler",
-  "source_url": "http://projecteuler.net/problem=7"
+  "source_url": "https://projecteuler.net/problem=8"
```

Comment:
`problem=8` does not seem to be relevant here, I think `problem=7` should be kept

Reply:
That's a really good catch, thank you.

Reply:
I've submitted a PR upstream and pinged you as a reviewer. (We need 3 approvals to get PRs merged in the problem-specifications repository, so it might take a day to get to where I can fix this PR).

Reply:
I don't know if I have write access to that repository, so my approval might not count 😛

Reply:
> We need 3 approvals to get PRs merged in the problem-specifications repository

I know, I've been strategically avoiding making any PRs there if you haven't noticed already from our discussions.

Reply:
All maintainers have write-access to the repository, if I am not mistaken.

> I've been strategically avoiding making any PRs there

Ah, now that you mention it, I can see that. And given the history of the repository, I can empathize!